modules/API/API.js

```diff
@@ -208,6 +209,9 @@ function initCommands() {
             sendAnalytics(createApiEvent('raise-hand.toggled'));
             APP.store.dispatch(raiseHand(!raisedHand));
         },
+        'toggle-share-audio': () => {
+            APP.store.dispatch(startAudioScreenShareFlow());
```

Comment:
Can you send the `createApiEvent` analytics event here ? Also, shouldn't we check if audio-only screen share is supported ?

Reply:
added event and check.